src/reducers/rootReducer.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+
+const rootReducer = combineReducers({
+// dummy object for reducer
+  f: () => ({}),
```

Comment:
@mendozabree Maybe you an actual name that will be used later. Such as `user` or `articles`. `f` may not mean much.